### PR TITLE
[feat] Posts two new job openings

### PR DESCRIPTION
**Pull request by DLavin23**
@hatchloyalty/engineering 

- adds a link to each job post from the `/about` page
- adds a software engineer job opening
- adds a technical account manager opening

image

### CODE REVIEW

File: content/static-pages/jobs/software-engineer.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+title: "Software Engineer"
+---
+
+The Software Engineer builds and maintains the software that runs Hatch’s main customer engagement loyalty platform. This position works with other engineers, engineering management, product teams, and relevant business stakeholders to execute high-quality software and systems on a regular basis.
+
+## Primary responsibilities
+
+- Develop software through the entire lifecycle – from design, through authorship, testing, deployment, and maintenance
+
+- Produce systems and code that are clean and easy to maintain •Contribute to quality assurance via automated and manual testing
```
Comment (hatch-carl) [inline content/static-pages/jobs/software-engineer.md]: Was this supposed to be two bullet points?

File: content/static-pages/jobs/software-engineer.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- Identify and execute tasks as necessary to support desired outcomes for scheduled release cycles
+
+- Support fellow engineers and business team members to innovate and improve ways the Hatch business can be evolved using technology
+
+## Additional responsibilities
+
+- Review and debug code written by other engineers •Document systems and processes to support engineering operations
```
Comment (hatch-carl) [inline content/static-pages/jobs/software-engineer.md]: Again two bullet points?

File: content/static-pages/jobs/software-engineer.md (hunk 1 above)
Comment (DLavin23) [reply content/static-pages/jobs/software-engineer.md]: Nope, copy-paste error from the PDF. Nice catch.

File: content/static-pages/jobs/software-engineer.md (hunk 2 above)
Comment (DLavin23) [reply content/static-pages/jobs/software-engineer.md]: Nope, copy-paste error from the PDF. Nice catch.

**Pull request merged**